ccd-definition/CaseEventToFields/createOrder.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "createC21Order",
+    "CaseFieldID": "temporaryC21Order",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "COMPLEX",
+    "PageID": 1,
```

Comment:
I would like meaningful PageIDs, they don't need to be 1, 2, 3 etc... OrderInformation maybe?

Reply:
didn't know they could be meaningful - thanks